reconcile/utils/aws_api_typed/organization.py

```diff
@@ -93,12 +105,37 @@ def describe_create_account_status(
         )
         return AWSAccountStatus(**resp["CreateAccountStatus"])
 
-    def move_account(
-        self, account_id: str, source_parent_id: str, destination_parent_id: str
-    ) -> None:
+    def get_ou(self, uid: str) -> str:
+        """Return the organizational unit ID of an account."""
+        resp = self.client.list_parents(ChildId=uid)
+        for p in resp.get("Parents", []):
+            if p["Type"] in set(["ORGANIZATIONAL_UNIT", "ROOT"]):
```

Comment:
can use set literal directly to avoid convert list to set, `{"ORGANIZATIONAL_UNIT", "ROOT"}`

Reply:
Changed in 1866903f0c134ac7f9d8f6df26e7f3174ced5c38